lib/components/user/saved-trip-screen.js

```diff
@@ -93,45 +98,28 @@ class SavedTripScreen extends Component {
   _handleDeleteTrip = async () => {
     if (confirm('Would you like to remove this trip?')) {
       const { deleteUserMonitoredTrip } = this.props
-      const { monitoredTrip } = this.state
+      const monitoredTrip = this._getTripToEdit(this.props)
       await deleteUserMonitoredTrip(monitoredTrip.id)
 
       this._goToSavedTrips()
     }
   }
 
-  _handleSaveNewTrip = async () => {
-    await this._updateMonitoredTrip()
+  _handleSaveNewTrip = async monitoredTrip => {
+    await this._updateMonitoredTrip(monitoredTrip)
     this._goToTripPlanner()
```

Comment:
After using this feature for a bit, I think that perhaps we should redirect users to the list of saved trips instead of sending them back to the trip planner here.

Reply:
That would make sense.